service/resource/cloudformation/delete.go

```diff
@@ -0,0 +1,15 @@
+package cloudformation
+
+import (
+	"context"
+
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
+	return nil
+}
+
+func (r *Resource) NewDeletePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*framework.Patch, error) {
```

Comment:
Would have been cool to do the file separation in a separate PR. Reviews would way be faster on such "dumb" changes as opposed to mix it with more complex logical changes.